x/examples/outline-connectivity/main.go

```diff
@@ -89,6 +91,7 @@ func main() {
 	domainFlag := flag.String("domain", "example.com.", "Domain name to resolve in the test")
 	resolverFlag := flag.String("resolver", "8.8.8.8,2001:4860:4860::8888", "Comma-separated list of addresses of DNS resolver to use for the test")
 	protoFlag := flag.String("proto", "tcp,udp", "Comma-separated list of the protocols to test. Muse be \"tcp\", \"udp\", or a combination of them")
+	collectorFlag := flag.String("collector", "", "URL to send JSON error reports to")
```

Comment:
I think we can be a bit more explicit with the flag name, since `collector` doesn't convey what's being reported. For example:
```suggestion
	collectorFlag := flag.String("report-to", "", "URL to send JSON error reports to")
```

`report-collector` is long, and `report-to` is [what NEL uses](https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging#usage).